pygame/classify_capture.py

```diff
@@ -14,16 +14,61 @@
 
 """A demo to classify Pygame camera stream."""
 import argparse
-import os
-import io
-import time
+import collections
 from collections import deque
+import io
 import numpy as np
+import operator
+import os
 import pygame
 import pygame.camera
 from pygame.locals import *
+import tflite_runtime.interpreter as tflite
+import time
+
+Class = collections.namedtuple('Class', ['id', 'score'])
+EDGETPU_SHARED_LIB = 'libedgetpu.so.1'
+
+def make_interpreter(model_file):
+    model_file, *device = model_file.split('@')
+    return tflite.Interpreter(
+      model_path=model_file,
+      experimental_delegates=[
+          tflite.load_delegate(EDGETPU_SHARED_LIB,
+                               {'device': device[0]} if device else {})
+      ])
+
+def input_size(interpreter):
```

Comment:
Since you're not returning the full  tensor size but just the image size, i'd change the name of this helper to "input_image_size" or return the full tensor size